src/lib/receipt_chain_database_lib/database.ml

```diff
@@ -0,0 +1,253 @@
+open Core_kernel
+
+type ('key, 'value) tree_node =
+  | Root of {children: 'key list}
```

Comment:
@imeckler is there some cryptography reason we want to start the receipt_chain_hash field in the accounts at a random value?

Reply:
I'm pretty sure that each `receipt_chain_hash` starts initially at Receipt.Chain_hash.empty. Check out the code `genesis_ledger/functor.ml`